### PR TITLE
New dcat3  implementation report

### PR DESCRIPTION
**Pull request by riccardoAlbertoni**
Based on what we (Andrea in particular) prepared for DCAT 2, I have put together a first draft of  First Draft for DCAT 3 Implementation Report.

I would ask DCAT co-editors to take a look and start reviewing.

Preview: https://raw.githack.com/w3c/dxwg/newDcatImpementationReport/dcat3-implementation-report/index.html

### CODE REVIEW

**Comment by dr-shorthair** [comment]
Do you need reviews now in order to merge this early version of the implementation report into the main branch? 
Or do you want to wait until the implementation report is complete?

**Comment by riccardoAlbertoni** [comment]
> Do you need reviews now in order to merge this early version of the implementation report into the main branch? Or do you want to wait until the implementation report is complete?


I'm seeking initial feedback to gauge whether this Report fulfills its purpose or if any significant issues arise. 
Personally, I think this is a minimal implementation report but It might show enough for moving forward the REC.
I wonder if other editors see it differently. That is why I've submitted it as a Draft pull request and included all the editors as reviewers to ensure broad visibility and encourage their input.

File: dcat3-implementation-report/table-implementations.html [hunk 1]
```diff
@@ -1,89 +1,26 @@
...
+    <tr id="imp-REV10" class="">	<td>	<a href="#REV10">REV10</a>	</td><td>		</td><td>		</td><td>		</td><td><code>	dcat:Resource	</code></td><td><a data-cite="VOCAB-DCAT-3#Property:resource_has_current_version"><code>	dcat:hasCurrentVersion	</code></a></td><td>		</td><td class="nr success">	1	</td><td class="nr warn">	0	</td><td>	</td><td></td><td>&#x2714;</td><td>	</td>	</tr>
+    <tr id="imp-REV11" class="">	<td>	<a href="#REV11">REV11</a>	</td><td>		</td><td>		</td><td>		</td><td><code>	dcat:Resource	</code></td><td><a data-cite="VOCAB-DCAT-3#Property:resource_version"><code>	dcat:version	</code></a></td><td>		</td><td class="nr success">	4	</td><td class="nr warn">	0	</td><td>	&#x2714;</td><td>(&#x2714;)</td><td>&#x2714;</td><td>&#x2714;	</td>	</tr>
+    <tr id="imp-REV12" class="">	<td>	<a href="#REV12">REV12</a>	</td><td>		</td><td>		</td><td>		</td><td><code>	dcat:Resource	</code></td><td><a data-cite="VOCAB-DCAT-3#Property:resource_version"><code>	adms:versiontes	</code></a></td><td>		</td><td class="nr success">	4	</td><td class="nr warn">	0	</td><td>	&#x2714;</td><td>&#x2714;</td><td>&#x2714;</td><td>&#x2714;	</td>	</tr>
```
Comment (andrea-perego) [inline dcat3-implementation-report/table-implementations.html]: |ERRATA | CORRIGE |
|--|--|
`adms:versiontes` | `adms:versionNotes`

**Review by andrea-perego — CHANGES_REQUESTED** [review]
Just a typo.

File: dcat3-implementation-report/index.html [hunk 2]
```diff
@@ -12,14 +12,170 @@
...
+          </thead>
+          <tbody>
+               <tr>
+              <td id="V02">V01</td>
+              <td>[[DCAT-AP-3]]</td>
+              <td>SEMIC action, Interoperable Europe, European Commission</td>
+              <td>Europe</td>
+              <td>2024</td>
+            </tr>
+              <tr>
```
Comment (andrea-perego) [inline dcat3-implementation-report/index.html]: I think DCAT-EP and GeoDCAT-AP has been swapped. I.e.:
- V02 should be GeoDCAT-AP
- V03 should be DCAT-EP

**Review by andrea-perego — CHANGES_REQUESTED** [review]
Vocabulary IDs to be fixed.

File: dcat3-implementation-report/index.html (hunk 2 above)
Comment (riccardoAlbertoni) [reply dcat3-implementation-report/index.html]: Thanks, Andrea 
https://github.com/w3c/dxwg/pull/1594/commits/97814c20884d189229588d36d37c9e312ae53bf4 should have fixed this.

File: dcat3-implementation-report/table-implementations.html (hunk 1 above)
Comment (riccardoAlbertoni) [reply dcat3-implementation-report/table-implementations.html]: Thanks Andrea, I have fixed this.

**Review by andrea-perego — APPROVED** [review]

**Review by davebrowning — APPROVED** [review]

**Pull request merged**